src/test/java/org/broadinstitute/consent/http/service/UseRestrictionConverterTest.java

```diff
@@ -24,7 +24,7 @@
 public class UseRestrictionConverterTest implements WithMockServer {
 
     private ClientAndServer mockServer;
-    private Integer port = 9000;
+    private Integer port = 9300;
```

Comment:
Unrelated to this PR, but Intellij hangs onto port 9000. This seems like a recent thing - I've run tests before with intellij up and this has always passed. With Intellij running:
```
$ lsof -nP -iTCP:9000 | grep LISTEN
idea    56681 grushton  975u  IPv6 0x48d22c605e90e3e5      0t0  TCP *:9000 (LISTEN)
```
With intellij down, nothing is listening:
```
$ lsof -nP -iTCP:9000 | grep LISTEN
```